content/shmem_reductions.tex

```diff
@@ -101,7 +106,7 @@
 %% C11
 {\color{Green}
 \begin{C11synopsis}
-int @\FuncDecl{shmem\_xor\_to\_all}@(TYPE *dest, const TYPE *source, int nreduce, shmem_team_t team);
+int @\FuncDecl{shmem\_xor\_reduce}@(shmem_team_t team, TYPE *dest, const TYPE *source, int nreduce;
```

Comment:
Missing a closing right-parenthesis here

Reply:
Fixed. Thank you.